src/OptimistAttestationResolver.sol

```diff
@@ -24,19 +22,19 @@ contract OptimistAttestationResolver is
     PausableUpgradeable,
     ReentrancyGuardUpgradeable
 {
-    event OptimistAttestationCreated(address indexed attester);
-    event OptimistAttestationRevoked(address indexed attester);
+    event AttesterAdded(address indexed attester);
+    event AttesterRemoved(address indexed attester);
```

Comment:
Clearer event names

Reply:
already have 
 /// @notice Emitted when an attester is allowed.
    event AttesterAllowed(address indexed attester);
    /// @notice Emitted when an attester is removed.
    event AttesterRemoved(address indexed attester);